src/scale.ts

```diff
@@ -170,8 +163,11 @@ export interface Scale {
   type?: ScaleType;
   /**
    * The domain of the scale, representing the set of data values. For quantitative data, this can take the form of a two-element array with minimum and maximum values. For ordinal/categorical data, this may be an array of valid input values.
+   *
+   * If the domain is 'pre-aggregation', we use the source data range as scale domain instead of aggregated data for aggregate axis.
+   * This property only works with aggregate functions that produce values within the raw data domain (`"mean"`, `"average"`, `"stdev"`, `"stdevp"`, `"median"`, `"q1"`, `"q3"`, `"min"`, `"max"`). For other aggregations that produce values outside of the raw data domain (e.g. `"count"`, `"sum"`), this property is ignored.
```

Comment:
Yep, and we have removed two of these from `SHARED_DOMAIN_OPS` already, so we just need to update comment here :)